packages/react/src/components/Table/Table.tsx

```diff
@@ -2,20 +2,27 @@ import React, { TableHTMLAttributes } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-const Table = ({
-  children,
-  className,
-  ...other
-}: TableHTMLAttributes<HTMLTableElement>) => (
-  <table className={classNames('Table', className)} {...other}>
+interface TableProps extends TableHTMLAttributes<HTMLTableElement> {
+  variant: 'border' | undefined;
+}
+
+const Table = ({ children, className, variant, ...other }: TableProps) => (
+  <table
+    className={classNames(
+      variant === 'border' ? 'Table Table--variant-border' : 'Table',
+      className
+    )}
```

Comment:
We've typically just been using the `--variantName` for our variant classes (see [Button](https://github.com/dequelabs/cauldron/blob/ded5fdafa720d19cc47b2e7b51f61437a37a3e6b/packages/styles/button.css#L23-L26) or [Tooltip](https://github.com/dequelabs/cauldron/blob/ded5fdafa720d19cc47b2e7b51f61437a37a3e6b/packages/styles/tooltip.css#L112) for examples).

```suggestion
    className={classNames(
      'Table',
      'Table--border': variant === 'border',
      className
    )}
```

Just be sure to remember to update the class name in css as well.

Reply:
Ah, okay. I'll make the changes.

Reply:
This exact syntax could not be used because `'Table--border': variant === 'border',` is not valid. Used `variant === 'border' && 'Table--border',` instead.

Reply:
Yup, that's my bad. What you have works but for FYI classnames also accepts object parameters and will apply the classnames based on the boolean values, so the following would also work:

```js
classNames(
  'Table',
  { 'Table--border': variant === 'border' },
  className
)
```